action.yml

```diff
@@ -25,26 +25,10 @@ inputs:
     description: 'In detached mode, the workflow job will continue while the tmate session is active'
     required: false
     default: 'false'
-  tmate-server-host:
-    description: 'The hostname for your tmate server (e.g. ssh.example.org)'
-    required: false
-    default: ''
-  tmate-server-port:
-    description: 'The port for your tmate server (e.g. 2222)'
-    required: false
-    default: ''
-  tmate-server-rsa-fingerprint:
-    description: 'The RSA fingerprint for your tmate server'
-    required: false
-    default: ''
-  tmate-server-ed25519-fingerprint:
-    description: 'The ed25519 fingerprint for your tmate server'
-    required: false
-    default: ''
```

Comment:
I'm wondering if we shouldn't keep support of these, but have a priority rule implementation (first take from action input, and if empty take from env, or the contrary). This will allow better compatibility with the upstream action behavior

Reply:
That is a good point, perhaps in a follow-up PR I can put them back in and have the priority rule implementation. Thank you for the suggestion! I'll create a ticket on this :)